zipline/data/data_portal.py

```diff
@@ -500,10 +500,10 @@ def get_single_asset_value(asset):
                      session_label > asset.end_date):
                 if field == "volume":
                     return 0
-                elif field != "last_traded":
-                    return np.NaN
                 elif field == "contract":
                     return None
+                elif field != "last_traded":
```

Comment:
Th `elif field == "contract"` wasn't getting hit because `elif field != "last_traded"` was `True`, so if `field` was "contract" this would return NaN instead of None.